content/applications/hr/employees/new_employee.rst

```diff
@@ -87,6 +95,8 @@ Optional fields
 Additional information tabs
 ===========================
 
+.. _employees/resume:
+
 Resumé tab
```

Comment:
I have updated the wording for the form, but I left the accent marks. The reason why, is Zac had made the edit originally in the ealrier version of these docs, and it appears in the live doc currently:
![image](https://github.com/odoo/documentation/assets/121518652/602a10c4-f132-41c3-9b7d-f39a82d21834)

So since it shows, and is more accurate, and Zac requested it originally, I am going to leave this for now.